tests/integ/test_workflow.py

```diff
@@ -1098,14 +1098,17 @@ def test_tuning(
         role=role,
         framework_version="1.5.0",
         py_version="py3",
-        instance_count=1,
-        instance_type="ml.m5.xlarge",
+        instance_count=instance_count,
```

Comment:
Where are these coming from? Do we have fixtures for `instance_count` and `instance_type` ?

Reply:
line 1093 and 1094, they are the pipeline parameters